Android/app/src/main/java/org/opendroneid/android/app/DebugActivity.java

```diff
@@ -63,6 +63,8 @@
 import java.util.Locale;
 import java.util.Set;
 
+import static org.opendroneid.android.BuildConfig.VERSION_NAME;
```

Comment:
I guess you intend this to be part of showing the version name as a toast? Right now this is not used anywhere though and clicking the About field doesn't do anything.

I think it would be better to keep the version number just shown at the bottom of the menu. Having it shown as a toast doesn't provide any additional usability benefit.